eq-author/src/App/page/Design/EditorLayout/Tabs/index.js

```diff
@@ -92,7 +92,7 @@ export const UnwrappedTabs = props => {
                 Component: Tab,
                 otherProps: { to: url(match), activeClassName },
               }
-            : { Component: DisabledTab, otherProps: {} };
+            : { Component: DisabledTab, otherProps: { disabled: true } };
```

Comment:
Surely a disabled tab should be disabled by default?

Reply:
I just looked into this. This prop is completely unnecessary. The DisabledTab is a span. Can we remove this and change the cypress test so that it checks that it becomes a link not loses the disabled attribute which wasn't doing anything.